app/javascript/src/javascripts/blacklists.js

```diff
@@ -41,16 +41,17 @@ Blacklist.parse_entries = function() {
 }
 
 Blacklist.toggle_entry = function(e) {
-  var tags = $(e.target).text();
+  var $link = $(e.target);
+  var tags = $link.text();
   var match = $.grep(Blacklist.entries, function(entry, i) {
     return entry.tags === tags;
   })[0];
   if (match) {
     match.disabled = !match.disabled;
     if (match.disabled) {
-      Blacklist.post_hide(e.target);
+      $link.addClass("blacklisted-inactive");
     } else {
-      Blacklist.post_unhide(e.target);
+      $link.removeClass("blacklisted-inactive");
```

Comment:
`post_hide` was called intentionally here to fix a problem with `video_with_sound` posts playing even when blacklisted (#2787). I'm not sure if this is still a problem.

Reply:
Ah, but it's not a post that's being passed in there, but the links in the blacklist menu. That wouldn't do anything to the video playing on the page regardless.